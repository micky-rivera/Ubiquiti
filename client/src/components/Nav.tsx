import React from 'react';
import Form from './Form';
import GridButton from './GridButton';
import Filter from './Filter';
import ListButton from './ListButton';

function Nav() {
  return (
    <div className="nav-bar">
        <Form />
        <div className='buttons'>
            <ListButton />
            <GridButton />
            <Filter />
        </div>
    </div>
  );
}

export default Nav;
