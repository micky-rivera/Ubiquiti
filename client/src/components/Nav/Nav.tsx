import React from 'react';
import Form from '../Form/Form';
import GridButton from '../GridButton/GridButton';
import Filter from '../Filter/Filter';
import ListButton from '../ListButton/ListButton';

function Nav() {
  return (
    <div className="nav-bar">
        <Form />
        <div className='buttons' data-testid='buttons'>
            <ListButton />
            <GridButton />
            <Filter />
        </div>
    </div>
  );
}

export default Nav;
