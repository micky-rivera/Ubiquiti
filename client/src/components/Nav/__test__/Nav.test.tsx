import React from "react";
import * as ReactDOM from "react-dom";
import Nav from '../Nav';
import { Provider } from 'react-redux'
import {store} from '../../../store/store'
import { render, cleanup } from "@testing-library/react";
import TestRenderer from 'react-test-renderer';

afterEach(cleanup);

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Provider store={store}><Nav /></Provider>, div)
})

it('renders the buttons', () => {
    const { getByTestId } = render(<Provider store={store}><Nav /></Provider>);
    expect(getByTestId('buttons')).toBeInTheDocument();
})

it('matches snapshot', () => {
    const tree = TestRenderer.create(<Provider store={store}><Nav /></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
})