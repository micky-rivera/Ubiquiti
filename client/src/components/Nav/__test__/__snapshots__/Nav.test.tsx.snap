// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="nav-bar"
>
  <svg
    className="search-form__svg"
    fill="none"
    height="20"
    viewBox="0 0 20 20"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M16.853 16.1465L14.006 13.3025C14.937 12.2115 15.5 10.7975 15.5 9.25049C15.5 5.79849 12.702 3.00049 9.25 3.00049C5.798 3.00049 3 5.79849 3 9.25049C3 12.7025 5.798 15.5005 9.25 15.5005C10.795 15.5005 12.208 14.9385 13.299 14.0095L16.147 16.8535C16.245 16.9515 16.372 16.9995 16.5 16.9995C16.628 16.9995 16.756 16.9505 16.854 16.8525C17.049 16.6585 17.049 16.3415 16.853 16.1465ZM4 9.25049C4 6.35549 6.355 4.00049 9.25 4.00049C12.145 4.00049 14.5 6.35549 14.5 9.25049C14.5 12.1455 12.145 14.5005 9.25 14.5005C6.355 14.5005 4 12.1455 4 9.25049Z"
      fill="#838691"
    />
  </svg>
  <form
    className="search-form"
    data-testid="search-form"
    onSubmit={[Function]}
  >
    <input
      className="search-form__input"
      onChange={[Function]}
      placeholder="Search"
      type="text"
      value=""
    />
  </form>
  <button
    className="search-form__clear-btn"
    data-testid="search-form__clear-btn"
    onClick={[Function]}
  >
    âœ•
  </button>
  <div
    className="buttons"
    data-testid="buttons"
  >
    <div
      className="hidden"
    >
      <svg
        className="list-btn__svg"
        data-testid="btn-inactive"
        fill="none"
        height="20"
        onClick={[Function]}
        viewBox="0 0 21 20"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <mask
          fill="white"
          id="path-1-inside-1_20_31063"
        >
          <path
            clipRule="evenodd"
            d="M6.5 4.5C6.5 5.32843 5.82843 6 5 6C4.17157 6 3.5 5.32843 3.5 4.5C3.5 3.67157 4.17157 3 5 3C5.82843 3 6.5 3.67157 6.5 4.5ZM8.25 3.75H16.75C17.164 3.75 17.5 4.086 17.5 4.5C17.5 4.914 17.164 5.25 16.75 5.25H8.25C7.836 5.25 7.5 4.914 7.5 4.5C7.5 4.086 7.836 3.75 8.25 3.75ZM5 11.5C5.82843 11.5 6.5 10.8284 6.5 10C6.5 9.17157 5.82843 8.5 5 8.5C4.17157 8.5 3.5 9.17157 3.5 10C3.5 10.8284 4.17157 11.5 5 11.5ZM8.25 9.25H16.75C17.164 9.25 17.5 9.586 17.5 10C17.5 10.414 17.164 10.75 16.75 10.75H8.25C7.836 10.75 7.5 10.414 7.5 10C7.5 9.586 7.836 9.25 8.25 9.25ZM5 17C5.82843 17 6.5 16.3284 6.5 15.5C6.5 14.6716 5.82843 14 5 14C4.17157 14 3.5 14.6716 3.5 15.5C3.5 16.3284 4.17157 17 5 17ZM8.25 14.75H16.75C17.164 14.75 17.5 15.086 17.5 15.5C17.5 15.914 17.164 16.25 16.75 16.25H8.25C7.836 16.25 7.5 15.914 7.5 15.5C7.5 15.086 7.836 14.75 8.25 14.75Z"
            fillRule="evenodd"
          />
        </mask>
        <path
          d="M5 7C6.38071 7 7.5 5.88071 7.5 4.5H5.5C5.5 4.77614 5.27614 5 5 5V7ZM2.5 4.5C2.5 5.88071 3.61929 7 5 7V5C4.72386 5 4.5 4.77614 4.5 4.5H2.5ZM5 2C3.61929 2 2.5 3.11929 2.5 4.5H4.5C4.5 4.22386 4.72386 4 5 4V2ZM7.5 4.5C7.5 3.11929 6.38071 2 5 2V4C5.27614 4 5.5 4.22386 5.5 4.5H7.5ZM16.75 2.75H8.25V4.75H16.75V2.75ZM18.5 4.5C18.5 3.53372 17.7163 2.75 16.75 2.75V4.75C16.6117 4.75 16.5 4.63828 16.5 4.5H18.5ZM16.75 6.25C17.7163 6.25 18.5 5.46628 18.5 4.5H16.5C16.5 4.36172 16.6117 4.25 16.75 4.25V6.25ZM8.25 6.25H16.75V4.25H8.25V6.25ZM6.5 4.5C6.5 5.46628 7.28372 6.25 8.25 6.25V4.25C8.38828 4.25 8.5 4.36172 8.5 4.5H6.5ZM8.25 2.75C7.28372 2.75 6.5 3.53372 6.5 4.5H8.5C8.5 4.63828 8.38828 4.75 8.25 4.75V2.75ZM5.5 10C5.5 10.2761 5.27614 10.5 5 10.5V12.5C6.38071 12.5 7.5 11.3807 7.5 10H5.5ZM5 9.5C5.27614 9.5 5.5 9.72386 5.5 10H7.5C7.5 8.61929 6.38071 7.5 5 7.5V9.5ZM4.5 10C4.5 9.72386 4.72386 9.5 5 9.5V7.5C3.61929 7.5 2.5 8.61929 2.5 10H4.5ZM5 10.5C4.72386 10.5 4.5 10.2761 4.5 10H2.5C2.5 11.3807 3.61929 12.5 5 12.5V10.5ZM16.75 8.25H8.25V10.25H16.75V8.25ZM18.5 10C18.5 9.03372 17.7163 8.25 16.75 8.25V10.25C16.6117 10.25 16.5 10.1383 16.5 10H18.5ZM16.75 11.75C17.7163 11.75 18.5 10.9663 18.5 10H16.5C16.5 9.86171 16.6117 9.75 16.75 9.75V11.75ZM8.25 11.75H16.75V9.75H8.25V11.75ZM6.5 10C6.5 10.9663 7.28372 11.75 8.25 11.75V9.75C8.38828 9.75 8.5 9.86171 8.5 10H6.5ZM8.25 8.25C7.28372 8.25 6.5 9.03372 6.5 10H8.5C8.5 10.1383 8.38828 10.25 8.25 10.25V8.25ZM5.5 15.5C5.5 15.7761 5.27614 16 5 16V18C6.38071 18 7.5 16.8807 7.5 15.5H5.5ZM5 15C5.27614 15 5.5 15.2239 5.5 15.5H7.5C7.5 14.1193 6.38071 13 5 13V15ZM4.5 15.5C4.5 15.2239 4.72386 15 5 15V13C3.61929 13 2.5 14.1193 2.5 15.5H4.5ZM5 16C4.72386 16 4.5 15.7761 4.5 15.5H2.5C2.5 16.8807 3.61929 18 5 18V16ZM16.75 13.75H8.25V15.75H16.75V13.75ZM18.5 15.5C18.5 14.5337 17.7163 13.75 16.75 13.75V15.75C16.6117 15.75 16.5 15.6383 16.5 15.5H18.5ZM16.75 17.25C17.7163 17.25 18.5 16.4663 18.5 15.5H16.5C16.5 15.3617 16.6117 15.25 16.75 15.25V17.25ZM8.25 17.25H16.75V15.25H8.25V17.25ZM6.5 15.5C6.5 16.4663 7.28371 17.25 8.25 17.25V15.25C8.38829 15.25 8.5 15.3617 8.5 15.5H6.5ZM8.25 13.75C7.28371 13.75 6.5 14.5337 6.5 15.5H8.5C8.5 15.6383 8.38829 15.75 8.25 15.75V13.75Z"
          fill="#838691"
          mask="url(#path-1-inside-1_20_31063)"
        />
      </svg>
    </div>
    <div
      className="list-btn"
    >
      <svg
        className="list-btn__svg"
        data-testid="btn-active"
        fill="none"
        height="20"
        onClick={[Function]}
        viewBox="0 0 21 20"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0.5 4C0.5 1.79086 2.29086 0 4.5 0H16.5C18.7091 0 20.5 1.79086 20.5 4V16C20.5 18.2091 18.7091 20 16.5 20H4.5C2.29086 20 0.5 18.2091 0.5 16V4Z"
          fill="#F6F6F8"
        />
        <mask
          fill="white"
          id="path-2-inside-1_20_31110"
        >
          <path
            clipRule="evenodd"
            d="M6.5 4.5C6.5 5.32843 5.82843 6 5 6C4.17157 6 3.5 5.32843 3.5 4.5C3.5 3.67157 4.17157 3 5 3C5.82843 3 6.5 3.67157 6.5 4.5ZM8.25 3.75H16.75C17.164 3.75 17.5 4.086 17.5 4.5C17.5 4.914 17.164 5.25 16.75 5.25H8.25C7.836 5.25 7.5 4.914 7.5 4.5C7.5 4.086 7.836 3.75 8.25 3.75ZM5 11.5C5.82843 11.5 6.5 10.8284 6.5 10C6.5 9.17157 5.82843 8.5 5 8.5C4.17157 8.5 3.5 9.17157 3.5 10C3.5 10.8284 4.17157 11.5 5 11.5ZM8.25 9.25H16.75C17.164 9.25 17.5 9.586 17.5 10C17.5 10.414 17.164 10.75 16.75 10.75H8.25C7.836 10.75 7.5 10.414 7.5 10C7.5 9.586 7.836 9.25 8.25 9.25ZM5 17C5.82843 17 6.5 16.3284 6.5 15.5C6.5 14.6716 5.82843 14 5 14C4.17157 14 3.5 14.6716 3.5 15.5C3.5 16.3284 4.17157 17 5 17ZM8.25 14.75H16.75C17.164 14.75 17.5 15.086 17.5 15.5C17.5 15.914 17.164 16.25 16.75 16.25H8.25C7.836 16.25 7.5 15.914 7.5 15.5C7.5 15.086 7.836 14.75 8.25 14.75Z"
            fillRule="evenodd"
          />
        </mask>
        <path
          clipRule="evenodd"
          d="M6.5 4.5C6.5 5.32843 5.82843 6 5 6C4.17157 6 3.5 5.32843 3.5 4.5C3.5 3.67157 4.17157 3 5 3C5.82843 3 6.5 3.67157 6.5 4.5ZM8.25 3.75H16.75C17.164 3.75 17.5 4.086 17.5 4.5C17.5 4.914 17.164 5.25 16.75 5.25H8.25C7.836 5.25 7.5 4.914 7.5 4.5C7.5 4.086 7.836 3.75 8.25 3.75ZM5 11.5C5.82843 11.5 6.5 10.8284 6.5 10C6.5 9.17157 5.82843 8.5 5 8.5C4.17157 8.5 3.5 9.17157 3.5 10C3.5 10.8284 4.17157 11.5 5 11.5ZM8.25 9.25H16.75C17.164 9.25 17.5 9.586 17.5 10C17.5 10.414 17.164 10.75 16.75 10.75H8.25C7.836 10.75 7.5 10.414 7.5 10C7.5 9.586 7.836 9.25 8.25 9.25ZM5 17C5.82843 17 6.5 16.3284 6.5 15.5C6.5 14.6716 5.82843 14 5 14C4.17157 14 3.5 14.6716 3.5 15.5C3.5 16.3284 4.17157 17 5 17ZM8.25 14.75H16.75C17.164 14.75 17.5 15.086 17.5 15.5C17.5 15.914 17.164 16.25 16.75 16.25H8.25C7.836 16.25 7.5 15.914 7.5 15.5C7.5 15.086 7.836 14.75 8.25 14.75Z"
          fill="#838691"
          fillRule="evenodd"
        />
        <path
          d="M5 7C6.38071 7 7.5 5.88071 7.5 4.5H5.5C5.5 4.77614 5.27614 5 5 5V7ZM2.5 4.5C2.5 5.88071 3.61929 7 5 7V5C4.72386 5 4.5 4.77614 4.5 4.5H2.5ZM5 2C3.61929 2 2.5 3.11929 2.5 4.5H4.5C4.5 4.22386 4.72386 4 5 4V2ZM7.5 4.5C7.5 3.11929 6.38071 2 5 2V4C5.27614 4 5.5 4.22386 5.5 4.5H7.5ZM16.75 2.75H8.25V4.75H16.75V2.75ZM18.5 4.5C18.5 3.53372 17.7163 2.75 16.75 2.75V4.75C16.6117 4.75 16.5 4.63828 16.5 4.5H18.5ZM16.75 6.25C17.7163 6.25 18.5 5.46628 18.5 4.5H16.5C16.5 4.36172 16.6117 4.25 16.75 4.25V6.25ZM8.25 6.25H16.75V4.25H8.25V6.25ZM6.5 4.5C6.5 5.46628 7.28372 6.25 8.25 6.25V4.25C8.38828 4.25 8.5 4.36172 8.5 4.5H6.5ZM8.25 2.75C7.28372 2.75 6.5 3.53372 6.5 4.5H8.5C8.5 4.63828 8.38828 4.75 8.25 4.75V2.75ZM5.5 10C5.5 10.2761 5.27614 10.5 5 10.5V12.5C6.38071 12.5 7.5 11.3807 7.5 10H5.5ZM5 9.5C5.27614 9.5 5.5 9.72386 5.5 10H7.5C7.5 8.61929 6.38071 7.5 5 7.5V9.5ZM4.5 10C4.5 9.72386 4.72386 9.5 5 9.5V7.5C3.61929 7.5 2.5 8.61929 2.5 10H4.5ZM5 10.5C4.72386 10.5 4.5 10.2761 4.5 10H2.5C2.5 11.3807 3.61929 12.5 5 12.5V10.5ZM16.75 8.25H8.25V10.25H16.75V8.25ZM18.5 10C18.5 9.03372 17.7163 8.25 16.75 8.25V10.25C16.6117 10.25 16.5 10.1383 16.5 10H18.5ZM16.75 11.75C17.7163 11.75 18.5 10.9663 18.5 10H16.5C16.5 9.86171 16.6117 9.75 16.75 9.75V11.75ZM8.25 11.75H16.75V9.75H8.25V11.75ZM6.5 10C6.5 10.9663 7.28372 11.75 8.25 11.75V9.75C8.38828 9.75 8.5 9.86171 8.5 10H6.5ZM8.25 8.25C7.28372 8.25 6.5 9.03372 6.5 10H8.5C8.5 10.1383 8.38828 10.25 8.25 10.25V8.25ZM5.5 15.5C5.5 15.7761 5.27614 16 5 16V18C6.38071 18 7.5 16.8807 7.5 15.5H5.5ZM5 15C5.27614 15 5.5 15.2239 5.5 15.5H7.5C7.5 14.1193 6.38071 13 5 13V15ZM4.5 15.5C4.5 15.2239 4.72386 15 5 15V13C3.61929 13 2.5 14.1193 2.5 15.5H4.5ZM5 16C4.72386 16 4.5 15.7761 4.5 15.5H2.5C2.5 16.8807 3.61929 18 5 18V16ZM16.75 13.75H8.25V15.75H16.75V13.75ZM18.5 15.5C18.5 14.5337 17.7163 13.75 16.75 13.75V15.75C16.6117 15.75 16.5 15.6383 16.5 15.5H18.5ZM16.75 17.25C17.7163 17.25 18.5 16.4663 18.5 15.5H16.5C16.5 15.3617 16.6117 15.25 16.75 15.25V17.25ZM8.25 17.25H16.75V15.25H8.25V17.25ZM6.5 15.5C6.5 16.4663 7.28371 17.25 8.25 17.25V15.25C8.38829 15.25 8.5 15.3617 8.5 15.5H6.5ZM8.25 13.75C7.28371 13.75 6.5 14.5337 6.5 15.5H8.5C8.5 15.6383 8.38829 15.75 8.25 15.75V13.75Z"
          fill="#838691"
          mask="url(#path-2-inside-1_20_31110)"
        />
      </svg>
    </div>
    <div
      className="grid-btn"
    >
      <svg
        className="grid-btn__svg"
        fill="none"
        height="20"
        onClick={[Function]}
        viewBox="0 0 21 20"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M9 8.5V4H4.5V8.5H9ZM4.5 3H9C9.552 3 10 3.448 10 4V8.5C10 9.052 9.552 9.5 9 9.5H4.5C3.948 9.5 3.5 9.052 3.5 8.5V4C3.5 3.448 3.948 3 4.5 3ZM9 16V11.5H4.5V16H9ZM4.5 10.5H9C9.552 10.5 10 10.948 10 11.5V16C10 16.552 9.552 17 9 17H4.5C3.948 17 3.5 16.552 3.5 16V11.5C3.5 10.948 3.948 10.5 4.5 10.5ZM16.5 4V8.5H12V4H16.5ZM16.5 3H12C11.448 3 11 3.448 11 4V8.5C11 9.052 11.448 9.5 12 9.5H16.5C17.052 9.5 17.5 9.052 17.5 8.5V4C17.5 3.448 17.052 3 16.5 3ZM16.5 16V11.5H12V16H16.5ZM12 10.5H16.5C17.052 10.5 17.5 10.948 17.5 11.5V16C17.5 16.552 17.052 17 16.5 17H12C11.448 17 11 16.552 11 16V11.5C11 10.948 11.448 10.5 12 10.5Z"
          fill="#838691"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <div
      className="hidden"
    >
      <svg
        className="grid-btn__svg"
        fill="none"
        height="20"
        onClick={[Function]}
        viewBox="0 0 21 20"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0.5 4C0.5 1.79086 2.29086 0 4.5 0H16.5C18.7091 0 20.5 1.79086 20.5 4V16C20.5 18.2091 18.7091 20 16.5 20H4.5C2.29086 20 0.5 18.2091 0.5 16V4Z"
          fill="#F6F6F8"
        />
        <path
          clipRule="evenodd"
          d="M9 3C9.552 3 10 3.448 10 4V8.5C10 9.052 9.552 9.5 9 9.5H4.5C3.948 9.5 3.5 9.052 3.5 8.5V4C3.5 3.448 3.948 3 4.5 3H9ZM3.5 11.5C3.5 10.948 3.948 10.5 4.5 10.5H9C9.552 10.5 10 10.948 10 11.5V16C10 16.552 9.552 17 9 17H4.5C3.948 17 3.5 16.552 3.5 16V11.5ZM12 3C11.448 3 11 3.448 11 4V8.5C11 9.052 11.448 9.5 12 9.5H16.5C17.052 9.5 17.5 9.052 17.5 8.5V4C17.5 3.448 17.052 3 16.5 3H12ZM11 11.5C11 10.948 11.448 10.5 12 10.5H16.5C17.052 10.5 17.5 10.948 17.5 11.5V16C17.5 16.552 17.052 17 16.5 17H12C11.448 17 11 16.552 11 16V11.5Z"
          fill="#838691"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <p
      className="filter-btn"
      data-testid="filter-btn"
      onClick={[Function]}
    >
      Filter
    </p>
    <div
      className="hidden"
    >
      <div
        className="filter-popup__header"
      >
        <p
          className="filter-popup__title"
        >
          Filter
        </p>
        <p
          className="filter-popup__exit"
          onClick={[Function]}
        >
          âœ•
        </p>
      </div>
      <div
        className="filter-popup__body"
        data-testid="filter-popup__body"
      >
        <p
          className="filter-popup__section-title"
        >
          Product Line
        </p>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi"
          >
            UniFi
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi LTE"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi LTE"
          >
            UniFi LTE
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi LED"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi LED"
          >
            UniFi LED
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi Protect"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi Protect"
          >
            UniFi Protect
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi Access"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi Access"
          >
            UniFi Access
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UniFi Talk"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UniFi Talk"
          >
            UniFi Talk
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="AirMAX"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="AirMAX"
          >
            AirMAX
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="AirPrism"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="AirPrism"
          >
            AirPrism
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="EdgeMAX"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="EdgeMAX"
          >
            EdgeMAX
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="SunMAX"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="SunMAX"
          >
            SunMAX
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="AirFiber"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="AirFiber"
          >
            AirFiber
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UFiber"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UFiber"
          >
            UFiber
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="LTU"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="LTU"
          >
            LTU
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="AmpliFi"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="AmpliFi"
          >
            AmpliFi
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="mFi"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="mFi"
          >
            mFi
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UISP"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UISP"
          >
            UISP
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="UNMS"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="UISP"
          >
            UNMS
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="Smart"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="Smart"
          >
            Smart
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="SmartPower"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="SmartPower"
          >
            SmartPower
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="Security"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="Security"
          >
            Security
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="Conference"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="Conference"
          >
            Conference
          </label>
        </div>
        <div
          className="filter-popup__option"
        >
          <input
            checked={false}
            className="filter-popup__checkbox"
            id="Ubiquiti"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="filter-popup__label"
            htmlFor="Ubiquiti"
          >
            Ubiquiti
          </label>
        </div>
      </div>
    </div>
  </div>
</div>
`;
